front/src/applications/editor/components/EntityError.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import cx from 'classnames';
+import { BsFillExclamationOctagonFill } from 'react-icons/bs';
+import { uniqueId } from 'lodash';
+
+import { EditorEntity } from 'types';
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import { getInfraID } from 'reducers/osrdconf/selectors';
+import { InfraErrorLine } from './InfraErrors/InfraError';
+
+const EntityError: FC<{ entity: EditorEntity; className?: string }> = ({ entity, className }) => {
+  const { t } = useTranslation();
+  const infraID = useSelector(getInfraID);
+  const { data } = osrdEditoastApi.endpoints.getInfraByIdErrors.useQuery({
+    id: infraID || 0,
+    objectId: entity.properties.id,
+  });
```

Comment:
Better to skip the call if infraId is undefined I think
```suggestion
  const { data } = osrdEditoastApi.endpoints.getInfraByIdErrors.useQuery({
    id: infraID as number,
    objectId: entity.properties.id,
  }, { skip: !infraID);
```